WMF Framework/UIScrollView+Limits.swift

```diff
@@ -26,9 +26,7 @@ extension UIScrollView {
     }
 
     @objc public func wmf_setContentInset(_ updatedContentInset: UIEdgeInsets, scrollIndicatorInsets updatedScrollIndicatorInsets: UIEdgeInsets, preserveContentOffset: Bool = true) -> Bool {
-        guard updatedContentInset != contentInset || updatedScrollIndicatorInsets != scrollIndicatorInsets else {
```

Comment:
Question: any issues with this being removed? Was preventing the empty view from being sized correctly...

Reply:
Yes, it prevents duplicate content inset updates and duplicate calls to `scrollViewInsetsDidChange` - the empty view shouldn't have to rely on duplicated calls to either of those so it's likely a problem somewhere else

Reply:
Hmm also seeing the divider line under the search langs disappear after rotate... wonder if that's related...

Reply:
Looks like the divider disappearing is caused by `NavigationBarHider`'s `scrollViewDidScroll` setting `shadowAlpha` to 0 in its second `guard`. It seems like both lines in that `guard` which set `shadowAlpha` can be removed - fixes the bug without any side effects that I could see - I may be missing something...

Reply:
It looks like shadow hiding should be off on the search view altogether - it's there for views like explore & history where the nav bar shadow disappears. Can confirm with Carolyn